src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewIconCell.swift

```diff
@@ -7,13 +7,61 @@ import UIKit
 
 class DynamicTableViewIconCell: UITableViewCell {
 
+	// MARK: - Init
+
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
```

Comment:
Can't we just have a constant identifier? `DynamicTableViewIconCell.reuseIdentifier`

Reply:
Not sure what you mean, this is the designated initializer for UITableViewCell.